test/parallel/test-uv-errmap.js

```diff
@@ -3,22 +3,18 @@
 
 require('../common');
 const assert = require('assert');
-const {
-  getSystemErrorMap,
-  _errnoException
-} = require('util');
+const { getSystemErrorMap, _errnoException } = require('util');
 
 const { internalBinding } = require('internal/test/binding');
 const uv = internalBinding('uv');
 const uvKeys = Object.keys(uv);
 
 const errMap = getSystemErrorMap();
 
-uvKeys.forEach((key) => {
-  if (!key.startsWith('UV_'))
-    return;
+for (const key of uvKeys) {
+  if (!key.startsWith('UV_')) return;
```

Comment:
@dwitipandya93 The `return` statement makes sense in `forEach()`, but not in `for ... of`.

```suggestion
  if (!key.startsWith('UV_')) continue;
```

Reply:
yup, updated